pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractTopic.java

```diff
@@ -98,8 +99,7 @@ public AbstractTopic(String topic, BrokerService brokerService) {
         this.producers = new ConcurrentHashMap<>();
         this.isFenced = false;
         this.replicatorPrefix = brokerService.pulsar().getConfiguration().getReplicatorPrefix();
-        this.deleteWhileInactive =
-                brokerService.pulsar().getConfiguration().isBrokerDeleteInactiveTopicsEnabled();
+        this.inactiveTopicPolicies.setDeleteWhileInactive(brokerService.pulsar().getConfiguration().isBrokerDeleteInactiveTopicsEnabled());
```

Comment:
If the namespace level policy already specified and then restart the broker, the inactive topic policy should apply the namespace level policy.

Reply:
I have fixed it. If there is a namespace level policy, the policy will be restored from zk in the subclass construction method.